src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java

```diff
@@ -6,5 +6,9 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.imports.illegalimport;
+
 // xdoc section -- start
 // xdoc section -- end
+
```

Comment:
Please copy imports from other example classes.
Please pay attention to issue description images that shows how examples should be different from each other. It is ok to change them , they should be same except for trailing comments.

Reply:
done